src/main/api/2/studio-api-2.yaml

```diff
@@ -6168,6 +6295,36 @@ components:
                 type: string
                 description: The adjusted value if available
 
+    ClipboardItem:
+      type: object
+      properties:
+        path:
+          type: string
+          description: The path of the item
+        children:
+          type: array
+          description: The child items
+          items:
+            $ref: '#/components/schemas/ClipboardItem'
+      required:
+        - path
+
+    ClipboardRequest:
+      type: object
+      properties:
+        siteId:
+          type: string
+          description: The id of the site
+        targetPath:
+          type: string
+          description: The target path for the operation
+        item:
+          $ref: '#/components/schemas/ClipboardItem'
```

Comment:
See previous comment about hierarchy. Could we stick with a list of paths (strings)?